src/platform/silabs/rs911x/wfx_sl_ble_init.c

```diff
@@ -301,9 +307,7 @@ void rsi_ble_add_char_val_att(void * serv_handler, uint16_t handle, uuid_t att_t
     new_att.property = val_prop;
 
     if (data != NULL)
-    {
         memcpy(new_att.data, data, RSI_MIN(sizeof(new_att.data), data_len));
-    }

```

Comment:
small nit, leave the `{}` coding standard wise

Reply:
updated